packages/js/create-woo-extension/README.md

```diff
@@ -1,19 +1,37 @@
 # @woocommerce/create-woo-extension
 
-This is a template to be used with `@wordpress/create-block` to create a WooCommerce Extension starting point.
+Create Woo Extension scaffolds a fully functional modern development environment for integrating with WooCommerce. Use it to build extensions leveraging the power of WooCommerce.
 
-## Installation
+## Configuration Free
 
+With build tooling already configured, getting started developing modern Javascript screens requires no effort. Create Woo Extension adds a React page integrating with WooCommerce Admin. Also included are PHP and Javascript unit testing, linting, and Prettier IDE confguration for WooCommerce and WordPress.
+
+Add a project inside of your favorite environment setup or use the built in [wp-env](https://github.com/WordPress/gutenberg/tree/trunk/packages/env) for easily setting up a local WordPress environment for building and testing plugins and themes. `wp-env` is configured to load the latest WooCommerce to easily start developing with a single command.
+
+## Usage
+
+```
+npx @wordpress/create-block -t @woocommerce/create-woo-extension my-extension-name
 ```
-npx @wordpress/create-block -t @woocommerce/create-woo-extension
+
+Navigate to the newly created folder and get started.
+
+```
+cd my-extension-name
+npm install # Install dependencies
+npm run build # Build the javascript
+npm -g i @wordpress/env # If you don't already have wp-env
+wp-env start # Start Wordpress environment
 ```
 
-When this has completed, go to your WordPress plugins page and activate the plugin.
+Go to the WordPress plugins page and activate the plugin.
```

Comment:
I am pretty sure after running the `create-block` command, it is already activated. Do we still need this step?

Reply:
Thanks @roykho, you're right its already activated. I reworked the line a bit, hopefully that works better.